src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/spreadsheet/SupplierPricesSheet.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.spreadsheet
+
+import org.apache.poi.ss.usermodel.Workbook
+import uk.gov.justice.digital.hmpps.pecs.jpc.move.Move
+import uk.gov.justice.digital.hmpps.pecs.jpc.price.Price
+
+/**
+ * Sheet to represent the actual JPC supplier prices used for all of the calculations in the outputted spreadsheet.
+ */
+internal class SupplierPricesSheet(workbook: Workbook, header: Header) : PriceSheet(workbook.getSheet("JPC Price book")!!, header) {
+  override fun writeMove(move: Move) {}
+
+  fun writePrices(prices: List<Price>) {
```

Comment:
One thing I am aware of no ordering is applied when writing the prices to the spreadsheet.  So it is basically as they come.